src/elements/teaser-hero/teaser-hero.scss

```diff
@@ -10,13 +10,6 @@
   // which appears in normalize css of several frameworks.
   outline: none !important;
 
-  --sbb-teaser-image-brightness-hover: var(--sbb-hover-image-brightness);
-  --sbb-teaser-image-animation-duration: var(
-    --sbb-disable-animation-zero-duration,
-    var(--sbb-animation-duration-4x)
-  );
-  --sbb-teaser-image-animation-easing: var(--sbb-animation-easing);
-
```

Comment:
Can't the hover part also be moved to the core.scss ? (I think it works for all related components)
```scss
@include sbb.hover-mq($hover: true) {
  :host(:hover) {
    --sbb-teaser-image-brightness: var(--sbb-teaser-image-brightness-hover);
  }
}
```